packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -234,7 +234,7 @@ module.exports = function (log, config, bounces) {
     diners: 'Diners Club',
     discover: 'Discover',
     jcb: 'JCB',
-    mastercard: 'MasterCard',
+    mastercard: 'Mastercard',
```

Comment:
💯 Only spotted two instances of case-insensitive "Mastercard" in the codebase (as expected):

```sh
git grep -ioh "MaStErCaRd" packages | sort | uniq
# Mastercard
# mastercard
```

And the lowercase "mastercard" references don't look user facing, apart from the `alt=` tag on the accepted cards.

```sh
git grep -n " alt=" packages/fxa-payments-server/src/routes/Product/AcceptedCards/ | cat
```

https://github.com/mozilla/fxa/blob/88bcf995987df8c104cde2e083bd676be7e8a97c/packages/fxa-payments-server/src/routes/Product/AcceptedCards/index.tsx#L16-L19

<details>
<summary><kbd>git grep -n "mastercard" | wc -l # 11</kbd></summary>

```sh
packages/fxa-auth-server/docs/swagger/shared/descriptions.ts:26:    'Card brand (e.g. `amex`, `diners`, `discover`, `jcb`, `mastercard`, `unionpay`, `visa`, or `unknown`).',
packages/fxa-auth-server/lib/senders/email.js:237:    mastercard: 'Mastercard',
packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts:46:    cardType: 'mastercard',
packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts:80:    cardType: 'mastercard',
packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts:95:    cardType: 'mastercard',
packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts:111:      cardType: 'mastercard',
packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFirstInvoice/index.stories.ts:128:    cardType: 'mastercard',
packages/fxa-payments-server/src/components/PaymentProviderDetails/index.scss:51:.mastercard::before {
packages/fxa-payments-server/src/components/PaymentProviderDetails/index.scss:52:  background-image: url('../../images/mastercard.svg');
packages/fxa-payments-server/src/routes/Product/AcceptedCards/index.tsx:5:import mastercardLogo from '../../../images/mastercard.svg';
packages/fxa-payments-server/src/routes/Product/AcceptedCards/index.tsx:17:        <img src={mastercardLogo} alt="mastercard" />
```
</details>